android/src/main/java/com/reactnativestripesdk/StripeSdkModule.kt

```diff
@@ -21,6 +21,21 @@ import kotlinx.coroutines.runBlocking
 class StripeSdkModule(reactContext: ReactApplicationContext, cardFieldManager: StripeSdkCardViewManager) : ReactContextBaseJavaModule(reactContext) {
   private var cardFieldManager: StripeSdkCardViewManager = cardFieldManager
 
+  companion object {
+    private const val PAYMENT_REQUEST_CODE = 50000
+    private const val SETUP_REQUEST_CODE = 50001
+    private const val SOURCE_REQUEST_CODE = 50002
+    private const val ADD_PAYMENT_METHOD_REQUEST_CODE = 6001
+    private const val PAYMENT_SHEET_REQUEST_CODE = 6003
+    private const val PAYMENT_SHEET_OPTIONS_REQUEST_CODE = 6004
+
+    private val VALID_REQUEST_CODES = setOf(
```

Comment:
Are we concerned about these ever changing, no, right?

Reply:
ideally would be make them public in android-sdk to be able use it but for now it's marked as a private

Reply:
cc @michelleb-stripe ^